buku

```diff
@@ -5972,7 +5990,7 @@ POSITIONAL ARGUMENTS:
     browse.override_text_browser = False
 
     # Fallback to prompt if no arguments
-    if len(sys.argv) <= 2 and (len(sys.argv) == 1 or (len(sys.argv) == 2 and args.nostdin)):
+    if argv in ([], ['--nostdin']):
```

Comment:
(…What was the point of the 1st check if only 2 specific argument lists are allowed anyway? :sweat_smile:)

Reply:
…Incidentally, I believe this could be moved higher (possibly before the `piped_input()` call, thus also allowing for a simpler condition in there) :thinking: